dbuild/__init__.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+import argparse
+import sys
+import os
+import shutil
+import glob
+from dbuild import exceptions
+
+from jinja2 import Environment, FileSystemLoader
+from docker import Client
+from tempfile import mkdtemp
+
+def docker_client(url='unix://var/run/docker.sock'):
+    """ return docker client """
+    return Client(url)
+
+def build_image(docker_client, path, tag):
+    """ Build docker image"""
+    return ''.join(['\n'.join(line.values()) for line in docker_client.build(
```

Comment:
Then it should be like this

return '\n'.join((''.join(line.values()).strip() for line in docker_client.build(
        path=path, rm=True, forcerm=True, tag=tag, decode=True)))

without join for line.values, it says typeerror (please see below)

  File "/tmp/venv/src/dbuild/dbuild/__init__.py", line 20, in build_image
    path=path, rm=True, forcerm=True, tag=tag, decode=True))
TypeError: sequence item 0: expected string, list found